packages/core/src/event_delegation_utils.ts

```diff
@@ -100,12 +102,10 @@ export class GlobalEventDelegation {
   }
 
   removeEventListener(element: HTMLElement, eventName: string, callback: Function): void {
-    const newJsactionAttribute = element
-      .getAttribute(Attributes.JSACTION)
-      ?.split(';')
-      .filter((s) => s === eventName + ':')
-      .join(';');
-    element.setAttribute(Attributes.JSACTION, newJsactionAttribute ?? '');
+    const cache = getCache(element);
+    if (cache) {
+      delete cache[eventName];
```

Comment:
The `delete` is not recommended, since it makes subsequent property reads slower. Can we do this instead?
```suggestion
      cache[eventName] = undefined;
```

Reply:
done